addons/account/views/report_invoice.xml

```diff
@@ -19,30 +19,30 @@
                         <span t-field="o.number"/>
                     </h2>
 
-                    <div class="row mt32 mb32">
-                        <div class="col-auto" t-if="o.name" name="description">
+                    <div id="informations" class="row mt32 mb32">
+                        <div class="col-auto mt-2 mb-2" t-if="o.name" name="description">
                             <strong>Description:</strong>
-                            <p t-field="o.name"/>
+                            <p class="m-0" t-field="o.name"/>
                         </div>
-                        <div class="col-auto" t-if="o.date_invoice" name="invoice_date">
+                        <div class="col-auto mt-2 mb-2" t-if="o.date_invoice" name="invoice_date">
                             <strong>Invoice Date:</strong>
-                            <p t-field="o.date_invoice"/>
+                            <p class="m-0" t-field="o.date_invoice"/>
```

Comment:
When it is just about adding/removing margin, could you not do it in scss only ?

Reply:
Did we change our mind again on this? I already changed it to target 12 and tested the compatibility as best as I could.

Reply:
tbh, in my mind we decide to target saas 12.1 to be sure to don't break v12 report without update of xml, but with new css.

If you are "sure" (if you have tested) that it will work for existing database, let's be crazy and we can check it for v12.

Is it ready to merge, or you still have some small change to do ?